### PR TITLE
Add bootstrap-datetimepicker

### PR DESCRIPTION
**Pull request by 32graham**

### CODE REVIEW

File: package-overrides/github/Eonasdan/bootstrap-datetimepicker@4.7.14.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "format": "global",
+  "main": "build/js/bootstrap-datetimepicker.min.js",
```
Comment (guybedford) [inline package-overrides/github/Eonasdan/bootstrap-datetimepicker@4.7.14.json]: Can we use the unminified sources? It is good to do this for the best development experience.

File: package-overrides/github/Eonasdan/bootstrap-datetimepicker@4.7.14.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "registry": "github",
```
Comment (guybedford) [inline package-overrides/github/Eonasdan/bootstrap-datetimepicker@4.7.14.json]: Unnecessary.

File: package-overrides/github/Eonasdan/bootstrap-datetimepicker@4.7.14.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  "dependencies": {
+    "moment": "npm:moment",
+    "jquery": "github:components/jquery",
+    "bootstrap": "github:twbs/bootstrap@^3.0.0",
+    "css": "github:systemjs/plugin-css"
```
Comment (guybedford) [inline package-overrides/github/Eonasdan/bootstrap-datetimepicker@4.7.14.json]: If you did want to simplify the above, you can just write `css: '*', bootstrap: '^3.0.0', 'moment': '*', jquery: '*'` which is equalvent to the above.

**Comment by guybedford** [comment]
Apologies for missing the follow-up here. Do ping me when completing updates next time... github doesn't send notifications for new commits to PRs.

**Comment by 32graham** [comment]
:smile:  No worries. I forgot about this PR myself. We aren't using the bootstrap-datetimepicker anymore. Additionally my branch and the version the override was for were out of date anyway.

**Pull request closed without merging**